src/main/java/com/twilio/http/bearertoken/BearerTokenTwilioRestClient.java

```diff
@@ -103,9 +112,9 @@ public Response request(BearerTokenRequest request) {
         if (accessToken == null || accessToken.isEmpty()) {
             throw new AuthenticationException("Access Token can not be Null or Empty.");
         }
-        
+
         if (isAccessTokenExpired()) {
```

Comment:
Rename it, isTokenExpired() and pass token as a argument.
In future we might be checking for refreshToken as well